main.go

```diff
@@ -82,7 +83,12 @@ func main() {
 				SupportedServers: []string{"primary"}}.Annotated(),
 			arrange.UnmarshalKey("prometheus", touchstone.Config{}),
 			arrange.UnmarshalKey("prometheus.handler", touchhttp.Config{}),
+			arrange.UnmarshalKey("onErrorHTTPResponse", OnErrorHTTPResponseOption{}),
```

Comment:
Given the below code, it looks like the default if this is unset is `Bearer`.  I think it's possible to  set the `OnErrorHTTPResponseOption.AuthType` in the object here? Then if it's not set in configuration, the default value is still there.  Then we wouldn't need `GetAuthType()`.